tests/ui/implied-bounds/normalization-nested-no-compat.rs

```diff
@@ -0,0 +1,42 @@
+// Test for normalization of projections that appear in the item bounds
+// (versus those that appear directly in the input types).
+//
+// FIXME(-Zno-implied-bounds-compat): remove this test once this flag is removed,
+// duplicate of `normalization-nested.rs`.
```

Comment:
Can you just make 4 revisions in `normalization-nested`? lifetime or param x compat or non-comat?